src/librustc_target/spec/windows_gnu_base.rs

```diff
@@ -27,10 +29,7 @@ pub fn opts() -> TargetOptions {
             // And it seems that the linker fails to use import symbols from msvcrt
             // that are required from functions in msvcrt in certain cases. For example
             // `_fmode` that is used by an implementation of `__p__fmode` in x86_64.
-            // Listing the library twice seems to fix that, and seems to also be done
-            // by mingw's gcc (Though not sure if it's done on purpose, or by mistake).
-            //
-            // See https://github.com/rust-lang/rust/pull/47483
+            // The library is purposely listed twice to fix that.
```

Comment:
```suggestion
            // See https://github.com/rust-lang/rust/pull/47483 for some more details.
```
or
```suggestion
            // (see #47483 for some more details)
```
https://github.com/rust-lang/rust/pull/47483 does contain some additional information about this case, so it may make sense to leave the reference.